src/EnergyPlus/EconomicTariff.cc

```diff
@@ -482,11 +485,67 @@ namespace EconomicTariff {
 				tariff( iInObj ).energyConv = 9.4781712e-9;
 				tariff( iInObj ).demandConv = 0.00003412;
 			} else {
-				tariff( iInObj ).convChoice = conversionKWH;
-				tariff( iInObj ).energyConv = 0.0000002778;
-				tariff( iInObj ).demandConv = 0.001;
-				ShowWarningError( RoutineName + CurrentModuleObject + "=\"" + cAlphaArgs( 1 ) + "\" invalid data" );
-				ShowContinueError( cAlphaFieldNames( 3 ) + "=\"" + cAlphaArgs( 3 ) + "\", Defaulting to KWH." );
+
+
+				// TODO: is this needed?
```

Comment:
TODO? Multiple places. Is there something to do here or not?

Reply:
We need to add "m^3" as a choice in the IDD (in which case we might as well add gal and ft^3, perhaps kgal too, CCF is already included). (fyi, Most utilities I've deal with actually use HCF (=CCF) on their bill.)